### PR TITLE
Update JUnit, PowerMock, Mockito libraries

### PR DESCRIPTION
**Pull request by daniel-beck**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -151,24 +151,24 @@ THE SOFTWARE.
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.11</version>
+        <version>4.12</version>
```
Comment (KostyaSha) [inline pom.xml]: I feel that you just updated versions without trying to use it locally https://code.google.com/p/powermock/issues/detail?id=531 ?

Comment (daniel-beck) [inline pom.xml]: @KostyaSha Actually I did (well, `-Plight-test`), which is why there are other changes. Note the PR build's 0 test failures.

Comment (KostyaSha) [inline pom.xml]: Yes, it's strange. I fill that some plugins that will switch on this core may get some issues. But then can override dependencies.

**Comment by KostyaSha** [comment]
:+1: in case of problems you now know issue that i referenced :)

**Pull request merged**